src/gate/printer/graphml.cpp

```diff
@@ -0,0 +1,70 @@
+#include "graphml.h"
+
+namespace eda::printer::graphMl {
+
+std::string linkDescription(const Link &link) {
+	return std::to_string(link.source) + "_" + std::to_string(link.target);
```

Comment:
Use two spaces instead of tabs in this file. One should follow the code convention.